gtsfm/scene_optimizer.py

```diff
@@ -145,14 +146,24 @@ def create_computation_graph(
                 image_shape_graph[i1],
                 image_shape_graph[i2],
                 gt_i2Ti1,
+                gt_pose_i1,
+                gt_pose_i2,
+                gt_scene_mesh,
             )
 
+            # Store results.
             i2Ri1_graph_dict[(i1, i2)] = i2Ri1
             i2Ui1_graph_dict[(i1, i2)] = i2Ui1
             v_corr_idxs_graph_dict[(i1, i2)] = v_corr_idxs
-
             two_view_reports_dict[(i1, i2)] = two_view_report
 
+            # # Use ground truth relative poses.
+            # from gtsam import Unit3
```

Comment:
looks like this should be removed before we check it in

Reply:
removed